coduno.go

```diff
@@ -146,7 +146,7 @@ func setupBaseHandler(w http.ResponseWriter, r *http.Request) context.Context {
 // API_TOKEN required for api calls
 func initSession(w http.ResponseWriter, r *http.Request) {
 	session, _ := cs.New(r, sessionID)
-	session.Options.MaxAge = 12 * 3600
+	session.Options.MaxAge = 12 * 3600 * 1000
```

Comment:
[Are you mad?](https://www.google.com/webhp?#q=12%20*%203600%20*%201000%20seconds)